pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -201,6 +201,14 @@ type CephStatus struct {
 	LastChecked    string                       `json:"lastChecked,omitempty"`
 	LastChanged    string                       `json:"lastChanged,omitempty"`
 	PreviousHealth string                       `json:"previousHealth,omitempty"`
+	Capacity       Capacity                     `json:"Capacity,omitempty"`
+}
+
+type Capacity struct {
+	TotalBytes     uint64 `json:"bytesTotal,omitempty"`
+	UsedBytes      uint64 `json:"bytesUsed,omitempty"`
+	AvailableBytes uint64 `json:"bytesAvail,omitempty"`
```

Comment:
It's better to spell it out
```suggestion
	AvailableBytes uint64 `json:"bytesAvailable,omitempty"`
```

Reply:
Done.